### PR TITLE
[terra-scroll] Added realistic examples 

### PR DESCRIPTION
**Pull request by AH106586Harika**
### Summary

**What was changed:**
updated 'terra-scroll' to add realistic content with interactive elements.

**Why it was changed:**
As the examples are generic changed to realistic examples.

### After Implementation

image

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [x] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

**This PR resolves:**

[UXPLATFORM-9462](https://jira2.cerner.com/browse/UXPLATFORM-9462)

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-core-docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased
+* Updated
+  * Updated `terra-scroll` to add realistic examples.
```
Comment (supreethmr) [inline packages/terra-core-docs/CHANGELOG.md:5]:
```suggestion

* Updated
  * Updated `terra-scroll` to add realistic examples.
```

**Review by adoroshk — COMMENTED** [review]
Before the change the vertical scroll example had only vertical scroll and horizontal scroll example had only horizontal scroll. When I tested this change, both examples had both, horizontal and vertical scrolls. It might be an expected behavior for the horizontal scroll in case of content being bigger than container, but doesn't look like expected behavior for the vertical scroll container to have a horysontal scroll. Please correct me if I am wrong. 

Also it might look better if the input and button have same padding ad paragraph for consistancy.

Screenshot 2023-08-14 at 9 45 05 AM
Screenshot 2023-08-14 at 9 48 18 AM

File: packages/terra-core-docs/CHANGELOG.md (hunk 1 above)
Comment (AH106586Harika) [reply packages/terra-core-docs/CHANGELOG.md:6]: Done.

**Comment by AH106586Harika** [comment]
> Before the change the vertical scroll example had only vertical scroll and horizontal scroll example had only horizontal scroll. When I tested this change, both examples had both, horizontal and vertical scrolls. It might be an expected behavior for the horizontal scroll in case of content being bigger than container, but doesn't look like expected behavior for the vertical scroll container to have a horysontal scroll. Please correct me if I am wrong.
> 
> Also it might look better if the input and button have same padding ad paragraph for consistancy.
> 
> Screenshot 2023-08-14 at 9 45 05 AM Screenshot 2023-08-14 at 9 48 18 AM Before the change the vertical scroll example had only vertical scroll and horizontal scroll example had only horizontal scroll. When I tested this change, both examples had both, horizontal and vertical scrolls. It might be an expected behavior for the horizontal scroll in case of content being bigger than container, but doesn't look like expected behavior for the vertical scroll container to have a horysontal scroll. Please correct me if I am wrong.
> 
> Also it might look better if the input and button have same padding ad paragraph for consistancy.
> 
> Screenshot 2023-08-14 at 9 45 05 AM Screenshot 2023-08-14 at 9 48 18 AM



Removed horizontal scroll for vertical scroll and reduced content, also removed vertical scroll for horizontal scroll, added padding as suggested for input and submit

**Review by adoroshk — CHANGES_REQUESTED** [review]
I suggest simplifying the `Scroll` content to be:

<div className={cx('paragraph-x-wrapper')}>
    <p>
      Entering patient details is a crucial process in healthcare facilities that involves capturing and documenting relevant information about
      individuals seeking medical care. Patient details need to be regularly updated to ensure accuracy and relevancy. This may involve periodic
      verification of personal information, updating medical history, and making necessary changes to insurance or contact details.
    </p>
    <InputField
        inputId="input-field"
        label="Enter Patient Name"
        type="text"
    />
    <Button text="Submit" />
</div>

In this case we don't need `container-x-wrapper` class.

**Review by supreethmr — APPROVED** [review]

**Review by sdadn — APPROVED** [review]

**Comment by kademswetha** [comment]
+1

**Pull request merged**